pkg/controller/containerjfr/containerjfr_controller.go

```diff
@@ -277,6 +277,36 @@ func (r *ReconcileContainerJFR) Reconcile(request reconcile.Request) (reconcile.
 		return reconcile.Result{}, err
 	}
 
+	// Check that secrets mounted in /truststore conincide with CRD
```

Comment:
s/conincide/coincide

Reply:
Im not sure what s/coincide/coincide means?

Reply:
Spelling mistake, change conincide to coincide. It's `sed` syntax :smile: